libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts

```diff
@@ -42,6 +42,29 @@ export const parseJsonOperation = {
   },
 };
 
+export const chunktext = {
+  name: 'chunktext',
+  id: 'chunkText',
+  type: 'Chunktext',
+  properties: {
+    api: {
+      id: 'connectionProviders/dataOperationNew',
+      name: 'dataOperationNew',
+      brandColor: '#8c6cff',
+      description: 'Data operations',
+      displayName: 'Data operations',
+      iconUri: 'https://logicappsv2resources.blob.core.windows.net/icons/compose.svg',
```

Comment:
Does this have compose icon only

Reply:
We are having a custom icon created, but for now it shares the same icon as all the other icons under Data Operations.